opentech/apply/funds/utils.py

```diff
@@ -0,0 +1,42 @@
+from opentech.apply.activity.messaging import messenger, MESSAGES
+from opentech.apply.users.models import User
+
+
+def can_alter_reviewers(instance, user):
+    return instance.stage.has_external_review and user == instance.lead
+
+
+def set_reviewers_fields(instance, user, fields):
```

Comment:
I don't like that this is doing three things.

1. get the submitted reviewers
2. excluding people who have already submitted
3. hiding fields that we dont need

I'm also not too keen for this to become divorced from the form as it naturally belongs there. 

maybe more like the following on the form class: (maybe `get_excluded_reviewers`) isnt needed...
```python
assigned_reviewers = instance.reviewers.all()
excluded_reviewers = get_excluded_reviewers(instance)
self.fields['staff_reviewers'] = populate_field(self.fields['staff_reviewers'], assigned_reviewers, excluded_reviewers)
if can_alter_reviewers(instance, user):
    self.fields['reviewer_reviewers'] = populate_field(self.fields['reviewer_reviewers'], assigned_reviewers, excluded_reviewers)
else:
    self.fields.pop('reviewer_reviewers')
```

Reply:
I moved it out of that form when I thought I would need to re-use it, and then I kept it out. I agree it should be moved back.